.github/workflows/deploy-frontend.yml

```diff
@@ -18,15 +18,22 @@ jobs:
 
   collect:
     runs-on: ubuntu-latest
+    if: ${{ always() }}
     needs: build-frontend
```

Comment:
As the docs say, collect uses the always() conditional expression so that it always runs after build-frontend has completed, regardless of whether it was successful.

Reply:
P.S. extremely sorry for these multiple pings 🙏

Reply:
This workflow worked on my fork 👍

Reply:
We don't want to continue if it fails though. Just if it succeeds or is skipped, no? We shouldn't try to deploy a known *failed* build.

Reply:
I came across these docs then https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

so instead of `${{ always() }}` should we do `${{ !failure() }}` ? Ig it should then work even if it was successful / skipped

Reply:
@Julian changing it to `${{ !failure() }}` also worked on my fork's workflow and didn't skip the `collect` job even when the `build-frontend` job was skipped.

Reply:
Cool, merging.then, let's see.

Reply:
After merge can you manually trigger the Collect new test reports workflow once ? so we can test out.